repos_info.json

```diff
@@ -33,6 +33,17 @@
       "web_application_type": "django",
       "versioning_strategy": "python"
     },
+    {
+      "name": "mit-open",
+      "repo_url": "https://github.com/mitodl/mit-open.git",
+      "ci_hash_url": "https://mit-open-ci.odl.mit.edu/static/hash.txt",
```

Comment:
I dunno if this will cause issues, but it looks like there's also no CI environment right now.

Should we remove `ci_hash_url`?

<img width="347" alt="Screenshot 2024-01-08 at 2 04 01 PM" src="https://github.com/mitodl/release-script/assets/9010790/a622889d-b393-4b0f-9169-3ac40f4183f6">

Reply:
@ChristopherChudzicki doof will crash if I leave that out